netci.groovy

```diff
@@ -49,168 +49,50 @@ build.cmd /no-node-reuse /no-deploy-extension /${configuration.toLowerCase()}
 // Add VSI jobs.
 // Generate the builds for commit and PRJob
 [true, false].each { isPR -> // Defines a closure over true and false, value assigned to isPR
-    def newVsiJobName = Utilities.getFullJobName(project, "vsi", isPR)
-
-    def newVsiJob = job(newVsiJobName) {
-        description('')
-
-        // This opens the set of build steps that will be run.
-        steps {
-            // Build roslyn-project-system repo - we also need to set certain environment variables for building the repo with VS15 toolset.
-            batchFile("""
-echo *** Build Roslyn Project System ***
-rmdir /S /Q %USERPROFILE%\\.nuget\\packages\\microbuild.plugins.swixbuild
-SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-
-build.cmd /no-node-reuse /release /skiptests
-""")
-
-            // Patch all the MSBuild xaml and targets files from the current roslyn-project-system commit into VS install.
-            batchFile("""
-echo *** Patch the MSBuild xaml and targets ***
```

Comment:
Deleting this code means that the just-built rules\targets wont take effect though right? I had set an env variable in launchSettings.json to help with F5 - those env vars should probably be set here as well